docs/_config.yml

```diff
@@ -0,0 +1,101 @@
+# Hexo Configuration
+## Docs: http://hexo.io/docs/configuration.html
+## Source: https://github.com/hexojs/hexo/
+
+# Site
+title: Apollo Link Docs
+propertytitle: Using Apollo Link
+subtitle: Apollo Link Guide
+description: A guide to using Apollo Link to customize Apollo Client.
+author:
+language:
+timezone:
+versions:
+  - '1'
+sidebar_categories:
+  null:
+    - index
+    - installation
+    - concepts
+  Usage:
+    - stateless
+    - stateful
+    - composition
```

Comment:
I think we could really use a `Recipes` section here, with stuff like:

- Setting headers
- Error handling
- Subscriptions

And maybe a `Links` section with all of the individual Link READMEs.

Reply:
I agree! I haven't gotten around to doing that yet but we should totally have one! cc @peggyrayzis